Makefile

```diff
@@ -91,6 +91,20 @@ generate/rust:
 
 	@echo Successfully compiled protos
 
+.PHONY: generate/python
```

Comment:
Should this target be added to `generate`? That's the target that executes all other generate targets.

Reply:
Good call, fixed